developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
+* the `aptos-core` repository checked out: `git clone https://github.com/aptos-labs/aptos-core.git`
+
+## Understand minting
+
+TODO: Consider moving all of this conceptual information to its own page, perhaps a new NFT index. Merge with conceptual information from Workshop #2.
+
+* Review the [mint_nft](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/mint_nft) source code and code comments within each subdirectory.
+* Explore the `mint_event_ticket` function defined within each subdirectory.
+
+### NFT types
+
+The two most common types of NFT are Event ticket / certificates and PFP NFTs.
+
+#### Event tickets and certificates
+
+This kind of NFT has a base token, and every new NFT generated from this base token has the same token data ID and image. They are generally used as certificates, meaning each NFT created from the base token is considered a printing edition of the base token.
+
+You might use this type of NFT for event tickets where each NFT is a ticket and has properties representing expiration date and if the ticket has been used. When you mint the NFT, you may set an expiration time for the event ticket and `is_ticket_used` to `false`. When the ticket is used, you update `is_ticket_used` to `true`.
+
+#### Profile picture (PFP) NFTs
+
+A PFP NFT has a unique token data ID and picture for each token. There are generally no printing editions of this NFT. Most NFT collections on NFT marketplaces are of this kind. They are generally proofs of ownership of an art piece.
+
+In this tutorial, we describe how to create and mint event ticket NFTs.
+
+### Accounts
+
+When you are minting an NFT, the NFT is stored under your [account](../../concepts/accounts.md) address. When you submit a transaction, you sign the transaction. Find your account configuration information in `.aptos/config.yaml` relative to where you run `aptos init` (below).
+
+[Resource accounts](../resource-accounts.md) allow the delegation of signing transactions. You create a resource account to grant a signer capability that can be stored in a new resource on the same account and can sign transactions autonomously. The signer capability is protected as no one has access to the private key for the resource account.
+
+### Initialization
+
+The `init_module` of [`minting.move`](https://github.com/aptos-labs/nft-tutorial/blob/main/sources/minting.move) always gets called and run when the module is published:
+
+```shell
+    fun init_module(resource_account: &signer) {
+        let resource_signer_cap = resource_account::retrieve_resource_account_cap(resource_account, @source_addr);
+        let resource_signer = account::create_signer_with_capability(&resource_signer_cap);
+```
+
+The `mint_nft` function of [`minting.move`](https://github.com/aptos-labs/nft-tutorial/blob/main/sources/minting.move) gets a collection and creates a token.
```

Comment:
could we update this link to the link in the public repo?

Reply:
I renamed to `mint_nft_ticket()` and removed the code link as this is going to a page on Move structure.